CHANGES.md

```diff
@@ -29,7 +31,7 @@ To be released.
      -  Added `Boolean.FalseFingerprint` static readonly field.
      -  Added `List.EmptyFingerprint` static readonly field.
      -  Added `Dictionary.EmptyFingerprint` static readonly field.
- -  Added `IValue.Type` property.  [[#50]]
+ -  Added `IValue.Kind` property.  [[#50]]
```

Comment:
I think this pull request should be also referenced as 16 LINE does.
```
 - Added `Bencodex.Types.ValueKind` enum type. [[#50], [#53]]
```

```suggestion
 -  Added `IValue.Kind` property.  [[#50], [#53]]
```

Reply:
Amended!